plugins/chain_plugin/include/eosio/chain_plugin/chain_plugin.hpp

```diff
@@ -262,6 +262,27 @@ class read_only {
 
    get_table_rows_result get_table_rows( const get_table_rows_params& params )const;
 
+   struct get_table_by_scope_params {
+      name        code; // mandatory
+      name        table = 0; // optional, act as filter
+      string      lower_bound; // lower bound of scope, optional
+      string      upper_bound; // upper bound of scope, optional
+      uint32_t    limit = 10;
+   };
+   struct get_table_by_scope_result_row {
+      name        code;
+      name        scope;
+      name        table;
+      name        payer;
+      uint32_t    count;
+   };
+   struct get_table_by_scope_result {
+      vector<get_table_by_scope_result_row> rows;
+      bool                more = false; ///< true if last element in data is not the end and sizeof data() < limit
```

Comment:
We want to move away from true/false for more. Instead fill it with the next value if there is more. See for example, get_producers or get_scheduled_transactions. Would be nice to also make that change for get table row as well.

Reply:
why not {next : "nextitem", more: 12345} or is there an other way to get the length of a table? I got referenced to this issue from here https://github.com/EOSIO/eos/issues/5543

Reply:
@kasperfish there's no way to know the remaining number unless you iterate all of them.